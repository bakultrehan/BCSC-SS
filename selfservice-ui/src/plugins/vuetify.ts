import Vue from 'vue';
import Vuetify from 'vuetify/lib';

Vue.use(Vuetify);

export default new Vuetify({
  icons: {
    iconfont: 'mdi',
  },
  theme: {
    themes: {
      light: {
        error: '#D8292F',
        success: '#2E8540',
        warning: '#FFC107',
      },
    },
    options: {
      cspNonce: 'cLv1G2y8KtMdR',
    },
  },
});
