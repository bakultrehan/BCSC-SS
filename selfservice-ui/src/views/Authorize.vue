<template>
  <v-container>
    <v-layout d-flex align-center justify-center row wrap>
      Authorizing...
    </v-layout>
  </v-container>
</template>
<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component({
  components: {}
})
export default class Authorize extends Vue {}
</script>
