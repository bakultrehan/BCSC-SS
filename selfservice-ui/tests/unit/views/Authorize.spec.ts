import { shallowMount, createLocalVue } from '@vue/test-utils';
import Authorize from '@/views/Authorize.vue';
import vuetify from 'vuetify';

describe('Authorize.vue', () => {
  it('renders without crashing', () => {
    const localVue = createLocalVue();
    localVue.use(vuetify);
    const wrapper = shallowMount(Authorize, {
      localVue
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
