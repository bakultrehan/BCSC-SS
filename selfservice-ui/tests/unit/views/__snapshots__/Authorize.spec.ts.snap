// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authorize.vue renders without crashing 1`] = `
<v-container-stub
  tag="div"
>
  <v-layout-stub
    align-center=""
    d-flex=""
    justify-center=""
    row=""
    tag="div"
    wrap=""
  >
    <div
      class="mt-10"
      col="4"
    >
      <v-progress-linear-stub
        active="true"
        buffervalue="100"
        class="loading"
        color="primary"
        height="6"
        indeterminate="true"
        rounded="true"
        value="0"
      />
      We know waiting sucks
      
      <br />
      but trust me itâ€™s worth it
    
    </div>
  </v-layout-stub>
</v-container-stub>
`;
